package.json

```diff
@@ -23,7 +23,7 @@
   },
   "devDependencies": {
     "coveralls": "^3.0.2",
-    "eslint": "^3.19.0",
+    "eslint": "^4.18.2",
```

Comment:
This mayor version change should be checked. Please refer to [Eslint - Migrating to v4.0.0
](https://eslint.org/docs/user-guide/migrating-to-4.0.0)